aws_topology/stackstate_checks/aws_topology/resources/sqs.py

```diff
@@ -4,8 +4,12 @@
 from schematics.types import StringType, ModelType
 
 
+def get_queue_name_from_url(url):
+    return url.rsplit('/', 1)[-1]
+
+
 def create_arn(region=None, account_id=None, resource_id=None, **kwargs):
-    return arn(resource='sqs', region=region, account_id=account_id, resource_id=resource_id)
+    return arn(resource='sqs', region='', account_id=account_id, resource_id=get_queue_name_from_url(resource_id))
```

Comment:
Is this a global resource or why was `region` dropped?

Reply:
This is actually fixed in #145 as it came up while I was changing the name of this component. There were issues with how SQS formats their URLs, which I resolved

Reply:
Merged in and fixed